io/plugins/eu.esdihumboldt.hale.io.xls/src/eu/esdihumboldt/hale/io/xls/AnalyseXLSSchemaTable.java

```diff
@@ -57,8 +57,7 @@ public AnalyseXLSSchemaTable(URI location, int sheetNum) throws Exception {
 	 */
 	@Override
 	protected void headerCell(int num, String text) {
-		if (num == header.size())
```

Comment:
With this check on and on all the examples I have tested, the check would fail if you would have at least one empty cell in a schema header, but as I added a counter in analysing the header the check can go back as it was.

Reply:
Ah ok, thanks for the explanation.